### PR TITLE
Removed all scratch org references

### PR DESCRIPTION
**Pull request by AnanyaJha**
### What does this PR do?

Updates all references to scratch org to be more generic.

### CODE REVIEW

File: packages/salesforcedx-vscode-core/package.nls.json [hunk 1]
```diff
@@ -34,13 +33,11 @@
   "force_lightning_event_create_text": "SFDX: Create Lightning Event",
   "force_lightning_interface_create_text": "SFDX: Create Lightning Interface",
   "force_source_status_local_text": "SFDX: View Local Changes",
-  "force_source_status_remote_text":
-    "SFDX: View Changes in Default Scratch Org",
+  "force_source_status_remote_text": "SFDX: View Changes in Default Org",
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode-core/package.nls.json]: I would suggest leaving "Scratch" in the messages on lines 10 through 36 (`force_source_pull_default_scratch_org_text` through `force_source_status_remote_text`). Scratch orgs are the only orgs with change tracking, so they're the only orgs these commands will work with.

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts [hunk 2]
```diff
@@ -50,28 +50,27 @@ export const messages = {
...
   parameter_gatherer_enter_project_name: 'Enter project name',
   parameter_gatherer_paste_forceide_url: 'Paste forceide:// URL from Setup',
   parameter_gatherer_paste_forceide_url_placeholder:
     'forceide:// URL from Setup',
   parameter_gatherer_invalid_forceide_url:
     "The forceide:// URL is invalid. From your subscriber's org, copy and paste the forceide:// URL shown on the Apex Debugger page in Setup.",
 
-  force_org_create_default_scratch_org_text:
-    'SFDX: Create a Default Scratch Org...',
+  force_org_create_default_scratch_org_text: 'SFDX: Create a Default Org...',
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode-core/src/messages/i18n.ts]: This command only works to create scratch orgs, so I'd leave it as is.

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts [hunk 3]
```diff
@@ -105,7 +103,7 @@ export const messages = {
   force_lightning_event_create_text: 'SFDX: Create Lightning Event',
   force_lightning_interface_create_text: 'SFDX: Create Lightning Interface',
   force_source_status_local_text: 'SFDX: View Local Changes',
-  force_source_status_remote_text: 'SFDX: View Changes in Default Scratch Org',
+  force_source_status_remote_text: 'SFDX: View Changes in Default Org',
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode-core/src/messages/i18n.ts]: The commands whose messages are on lines 66 through 106 (`force_source_pull_default_scratch_org_text` through `force_source_status_remote_text`)work only with scratch orgs, so I'd leave them unchanged.

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts [hunk 4]
```diff
@@ -58,19 +58,20 @@ export const messages = {
   parameter_gatherer_invalid_forceide_url:
     "The forceide:// URL is invalid. From your subscriber's org, copy and paste the forceide:// URL shown on the Apex Debugger page in Setup.",
 
-  force_org_create_default_scratch_org_text: 'SFDX: Create a Default Org...',
+  force_org_create_default_scratch_org_text:
+    'SFDX: Create a Default Scartch Org...',
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode-core/src/messages/i18n.ts]: There's a typo here: `Scartch` instead of `Scratch`.

Comment (AnanyaJha) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ts]: Just saw it, fixing!

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts (hunk 2 above)
Comment (AnanyaJha) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ts]: Fixing this as well.

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts [hunk 5]
```diff
@@ -50,7 +50,7 @@ export const messages = {
     'Enter desired directory (Press Enter to confirm or Esc to cancel)',
   parameter_gatherer_enter_username_name: 'Enter target username',
   parameter_gatherer_enter_alias_name:
-    'Enter an org alias or use default alias',
+    'Enter a scratch org alias or use default alias',
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode-core/src/messages/i18n.ts]: This one actually should be `an org alias`. This message is used both when you create scratch orgs and when you authorize any type of org.

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts [hunk 6]
```diff
@@ -59,9 +59,9 @@ export const messages = {
     "The forceide:// URL is invalid. From your subscriber's org, copy and paste the forceide:// URL shown on the Apex Debugger page in Setup.",
 
   force_org_create_default_scratch_org_text:
-    'SFDX: Create a Default Scartch Org...',
+    'SFDX: Create a Default Scratch Org...',
 
-  force_org_open_default_scratch_org_text: 'SFDX: Open Default Org',
+  force_org_open_default_scratch_org_text: 'SFDX: Open Default Scratch Org',
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode-core/src/messages/i18n.ts]: This should not contain `Scratch`. Any type of org can be your default org.

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts (hunk 5 above)
Comment (AnanyaJha) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ts]: Got it

File: packages/salesforcedx-vscode-core/src/messages/i18n.ts (hunk 6 above)
Comment (AnanyaJha) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ts]: Okay!

File: packages/salesforcedx-vscode-core/package.nls.json [hunk 7]
```diff
@@ -4,9 +4,8 @@
   "force_auth_web_login_authorize_dev_hub_text": "SFDX: Authorize a Dev Hub",
   "force_auth_web_login_authorize_org_text": "SFDX: Authorize an Org",
   "force_auth_logout_all_text": "SFDX: Log Out from All Authorized Orgs",
-  "force_org_create_default_scratch_org_text":
-    "SFDX: Create a Default Scratch Org...",
-  "force_org_open_default_scratch_org_text": "SFDX: Open Default Scratch Org",
+  "force_org_create_default_scratch_org_text": "SFDX: Create a Default Org...",
```
[the remark this answers is not in the document]
Comment (AnanyaJha) [reply packages/salesforcedx-vscode-core/package.nls.json]: No worries, I'll adjust it. :)

**Pull request merged**